app/test/v2/test_meetups.py

```diff
@@ -0,0 +1,47 @@
+"""test for api endpoints"""
+# inbuilt modelues
+import json
+import unittest
+# local imports
+from app import create_app
+class MeetupTest(unittest.TestCase):
+    def setUp(self):
+        self.appliaction = create_app
```

Comment:
Minor typo: `application`.
Also you are assigning a function reference and not actually the result of the function call

Reply:
I change typo on that in my next pr

Reply:
thanks for the feedback.
> Also you are assigning a function reference and not actually the result of the function call
 
i will work on this too